doc/dev/general.md

```diff
@@ -49,20 +49,34 @@ but only generic tools, i.e. `kubectl` and `helm`.
 the chosen foundation, and assuming that the cluster does not come
 with Helm Tiller pre-installed.
 
-### Deployment and Tear-down
+### Deployment and Tear-down - cf-operator
 
 ```shell
 helm install --name cf-operator \
      --namespace cfo \
      --set "global.operator.watchNamespace=kubecf" \
-     https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.1%2B92.g77e53fda.tgz
+     https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.2-147.gb88e4296.tgz
 ```
 
 In the example above, version 0.4.1 of the operator was used. Look
 into the `cf_operator` section of the top-level `def.bzl` file to find
 the version of the operator validated against the current kubecf
 master.
 
+**Note:**
+> The above `helm install`, will generate two controllers into `cfo` namespace. The first
+controller is the `cf-operator`, the second one is the `quarks-job`.
+>
+> The `global.operator.watchNamespace=kubecf` path, tells both
+controllers to watch for CRD´s instances into the `kubecf` namespace.
+>
+> The cf-operator will create the `kubecf` namespace, and use a webhook to label the namespace
+with the `cf-operator-ns` key.
+>
+> If the `kubecf` namespace is deleted, but the operators still running, they will not longer
+know which namespace to watch. This can lead to problems, so make sure you delete also the operators
```

Comment:
`delete also` -> `also delete`

Reply:
Maybe use `pods` instead of `operators` in this whole phrase?

Reply:
fixed